packages/itwinui-css/src/radio-tile/radio-tile.scss

```diff
@@ -59,22 +49,15 @@
   appearance: none;
   opacity: 0;
   position: absolute;
-  @include focus-visible {
-    + * {
-      outline-offset: -4px;
-      outline: 2px solid var(--iui-color-foreground-primary);
-      @media (forced-colors: active) {
-        outline-color: Highlight;
-      }
-    }
-  }
 
   &:checked + * {
-    padding: $iui-s - 1;
+    padding: $iui-s;
     z-index: 3;
-    border: 2px solid var(--iui-color-foreground-primary);
+    outline: 2px solid var(--iui-color-foreground-primary);
```

Comment:
minor, but could you variable `iui-xxs` for 2px

Reply:
I changed it in lines 56, 120, and 129-130. I went back and forth on leaving lines 129-130 as is or changing them. If you think it's better the other way let me know. (And also your reasoning.)

Reply:
yeah, leave 3px instead, agree